doc/api/stream.md

```diff
@@ -2413,7 +2413,8 @@ and `stream.Readable` classes, respectively. The `'finish'` event is emitted
 after [`stream.end()`][stream-end] is called and all chunks have been processed
 by [`stream._transform()`][stream-_transform]. The `'end'` event is emitted
 after all data has been output, which occurs after the callback in
-[`transform._flush()`][stream-_flush] has been called.
+[`transform._flush()`][stream-_flush] has been called. In the case of an error
+neither `'finish'` nor `'end'` *should* be emitted.
```

Comment:
Feel free to ignore this suggestion, but I like to avoid italics/bold/all-caps for emphasis where possible.

```suggestion
neither `'finish'` nor `'end'` should be emitted.
```